src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -293,8 +294,16 @@ public ObservableList<Task> getFilteredTaskList() {
         return filteredTasks;
     }
 
+    @Override
+    public ObservableList<Task> getTaskDueSoonList() {
+        FilteredList<Task> taskDueSoonList = new FilteredList<>(filteredTasks);
```

Comment:
I think the rationale of making a copy of the task list should be commented out since it is an unorthodox way to solve the problem.

Reply:
Yup, have commented it.